lib/mixins/listen-on-root.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Issue #569: collapse::toggle::state triggered too many times
+ * @link https://github.com/bootstrap-vue/bootstrap-vue/issues/569
+ */
+export default {
+    data() {
```

Comment:
A little more info on `_, $` prefixes and observation: https://github.com/vuejs/vue/issues/665#issuecomment-125265241